.github/workflows/ci-model-regression-on-schedule.yml

```diff
@@ -335,15 +335,14 @@ jobs:
         run: |-
           poetry run pip install analytics-python
           poetry run python .github/scripts/mr_publish_results.py
-          poetry run python .github/scripts/mr_generate_summary.py
           cat $SUMMARY_FILE
 
       - name: Upload an artifact with the report
```

Comment:
It seems that the `Download artifact` step is not needed anymore (in both workflows)